src/components/sidebar/Sidebar.jsx

```diff
@@ -1,17 +1,24 @@
 import React from "react";
 import "./sidebar.css";
+import { useNavigate } from "react-router-dom";
 
 function Sidebar() {
+  const navigate = useNavigate();
+
   return (
     <div>
       <div className="videolib-drawer-container">
         <div className="videolib-list-container">
           <ul className="videolib-list">
```

Comment:
suggestion: avoid hard coding, try making it generic and eliminate duplicate code